alchemy/src/util/telemetry/client.ts

```diff
@@ -96,6 +92,7 @@ export class TelemetryClient implements ITelemetryClient {
     if (events.length === 0) {
       return;
     }
+    const { userId, ...data } = this.context!;
```

Comment:
what happens if `this.context` is undefined? Why remove the early return?

Reply:
no clue, brought the early return back